### PR TITLE
Lint models/repo_branch.go

### PR DESCRIPTION
**Pull request by Bwko**

### CODE REVIEW

File: models/repo_branch.go [hunk 1]
```diff
@@ -34,24 +36,27 @@ func GetBranchesByPath(path string) ([]*Branch, error) {
...
+		return nil, &ErrBranchNotExist{branch}
 	}
 	return &Branch{
 		Path: repo.RepoPath(),
-		Name: br,
+		Name: branch,
 	}, nil
 }
 
+//GetBranches returns all the branches of a repository
```
Comment (lunny) [inline models/repo_branch.go]: space

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by strk** [comment]
LGTM

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**